integration/log_integration_test.go

```diff
@@ -276,6 +293,55 @@ func checkLogRootHashMatches(logID int64, tree *merkle.InMemoryMerkleTree, clien
 	return nil
 }
 
+// checkInclusionProofsAtIndex obtains and checks proofs at tree sizes from zero up to 2 x the sequencing
+// batch size (or number of leaves queued if less). The log should only serve proofs for indices in a tree
+// at least as big as the index where STHs where the index is a multiple of the sequencer batch size. All
+// proofs returned should match ones computed by the alternate Merkle Tree implementation, which differs
+// from what the log uses.
+func checkInclusionProofsAtIndex(testIndex int64, logID int64, tree *merkle.InMemoryMerkleTree, client trillian.TrillianLogClient, params testParameters) error {
+	for treeSize := int64(0); treeSize < min(params.leafCount, int64(2 * params.sequencerBatchSize)); treeSize++ {
+		ctx, cancelFunc := getRPCDeadlineContext()
```

Comment:
`cancelFunc` -> `cancel` 

(Go tends towards shorter names, and `cancel` is a common name here, https://blog.golang.org/context)